python/ray/tests/test_output.py

```diff
@@ -248,6 +251,7 @@ class A:
 
 
 # TODO(rickyx): Remove this after migration
+@skip_flaky_core_test_premerge("https://github.com/ray-project/ray/issues/43099")
```

Comment:
Why do we need to skip this test if, according to your Slack message, it doesn't fail? Is it also flaky?

Reply:
I thought I remembered seeing fail in one of the builds and it's the only other test in this file that uses autoscaler v2. Better to mark it flaky and avoid blocking more builds since fixing these should be prioritized in short order anyways.